lib/galaxy_test/api/test_roles.py

```diff
@@ -135,7 +139,7 @@ def test_create_only_admin(self):
         assert "administrator" in response_err["err_msg"]
 
     @staticmethod
-    def check_role_dict(role_dict, assert_id=None):
+    def check_role_dict(role_dict: dict, assert_id: Optional[str] = None):
```

Comment:
maybe `Dict`? Then we can subscribe it with key/val types for python 3.7)

Reply:
Good point, thanks! Fixed in a rebase.